package.json

```diff
@@ -1839,7 +1839,7 @@
                     "type": "webview",
                     "id": "jupyterViewVariables",
                     "name": "Variables",
-                    "when": "jupyter.isnativeactive"
+                    "when": "jupyter.isvscodenotebookactive"
```

Comment:
@DonJayamanne doesn't seem like I can get rid of this. Tried every variation of notebookViewType == jupyter-notebook here in this when, it never works and the view is never enabled.

Reply:
Same enablement does work fine for the commands, so I changed them.

Reply:
@DonJayamanne Sorry to bug, but can you take another look here? Would like to get this in and ported to release so that we can test in release.